operations/structure.md

```diff
@@ -0,0 +1,45 @@
+# Structure and responsibilities
+
+This page describes the structure and roles of our Product Team.
+
+## Responsibilities
+
+2i2c's Operatoins team is responsible for defining and overseeing our systems for work coordination, planning, and accountability.
+
+Below are major areas of responsibility.
+
+### Define systems and processes for coordination
+
+- Define our meeting and coordination systems for planning our work.
+- Define systems for sharing information between team members.
+
+### Refine work items to be actionable and clear
+
+- Translate strategic priorities and deliverables into actionable work items.
+- Ensure that our work items have a clear scope and path forward for completion.
+- Communicate with our Product and Engineering teams to align our teams on a plan.
```

Comment:
You are mentioning Prod and Eng here, you foresee Operations only connected to those areas or other areas as well??